### DISCUSSION TITLE
단축키가 생겼다!!!

### DISCUSSION THREAD

**Discussion by Skyler-act** [metamong-friends/metamong-friends.github.io#60] [제안하기]
마이크 단축키를 발견했는데 너무 좋아요!!ㅎㅎㅎ 
저의 개인적인 바람일 수도 있겠지만^^;; 아래 기능이 있다면 너무너무 좋을 것 같아서 살짝 제안드려 봅니다 ^^

단축키 안내 툴팁: 마이크 길게 호버시 툴팁 표시(예: shortcut- m)
추가 단축키: 처음 들어오자마자 하는일이 가운데 챗창과 왼쪽 접속자 리스트 창을 내리는 일인데, 한번에 촥 들어가는 단축키가 있으면 좋겠어요. 

메타몽 화이팅!!!

**Comment by prism27** [comment]
안녕하세요! TJ 입니다.

좋은 의견 너무 감사드립니다.
메타몽에 숨겨진 편의기능이 많이 생기는 상황인데, 어떻게 노출하면 좋을 지 
고민중에 있습니다.^^ 

추가 단축키 아이디어도 유저 커스텀 Action 을 통해 편하게 쓰시던 환경으로 세팅이 되거나 하면 좋을 것 같네요.
개선 아이디어로 저희도 관리하고 의견 나눠보도록 하겠습니다.

감사합니다!!
